documentation/slate/source/includes/interface.md

```diff
@@ -124,6 +96,74 @@ That means:
 * containing some [disruptions](#disruption) objects (**"type": "disruption"**)
 * where you can find the details of your object (**"id": "edc46f3a-ad3d-11e4-a5e1-005056a44da2"**).
 
+Paging
+------
+
+``` shell
+#Retrieving lines collection
+$ curl 'https://api.navitia.io/v1/coverage/sandbox/lines' -H 'Authorization: 3b036afe-0110-4202-b9ed-99718476c2e0'
+
+#You can find "paging" informations, as
+{
+    "pagination": {
+        "items_on_page": 25
+        "items_per_page": 25,
+        "start_page": 1,
+        "total_result": 1921,
+    },
+}
+
+#You can also find "paging" links, prebuilt in the link section
+{
+       {
+            "href": "https://api.navitia.io/v1/coverage/demo/lines?start_page=0",
+            "templated": false
+            "type": "previous",
+        },
+       {
+            "href": "https://api.navitia.io/v1/coverage/demo/lines?start_page=2",
+            "templated": false
+            "type": "next",
+        },
+       {
+            "href": "https://api.navitia.io/v1/coverage/demo/lines?start_page=76",
+            "templated": false
+            "type": "last",
+        },
+       {
+            "href": "https://api.navitia.io/v1/coverage/demo/lines",
+            "templated": false
+            "type": "first",
+        }
+}
+```
+
+
+<aside class="success">
+    The results are paginated to avoid crashing your parser. The parameters to get the next or previous page are within the `links` section of the result.
```

Comment:
Not mandatory:
```suggestion
    The results are paginated to avoid too big response size and crashing your parser. The parameters to get the next or previous page are within the `links` section of the result.
```